HumanResourceMachine/Interfaces/IBusinessLogicService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using HumanResourceMachine.Entities;
+
+namespace HumanResourceMachine.Interfaces
+{
+    public interface IBusinessLogicService
```

Comment:
business logic it is a layer, bot not the name of class or something else. for instance: u have 9999 classes which responsibility BL, they all will be BusinessLogicService1-9999 ? of course no

Reply:
Renamed